### ISSUE TITLE
can i send the backup created it and send it via email?

### ISSUE THREAD

**Issue by titosobabas** [lysender/dbautobackup#2]
can i send the backup created it and send it via email?

**Comment by linedotpaul** [comment]
I'm sure you could do if you used phpmailer for instance at the right point

**Comment by lysender** [comment]
Integrating it into the script would require more dependencies though. Maybe you can send it via the cron command instead?

I'll think about it. With composer, maybe I'll consider adding it.

**Comment by chope1** [comment]
Emailing the backups will be ideal.
Thanks.

**Comment by voltainc** [comment]
Yes you can, but it strictly depends on the size of your exported file, not recommended if it surpass designated file upload cap on most hosting.
*Not recommended
